### PR TITLE
Task solution

### PR DESCRIPTION
**Pull request by Oleh88777**
- [DEMO LINK](https://Oleh88777.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Oleh88777.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,78 @@
...
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+
+  background: #fff;
```
Comment (volodymyr-soltys97) [inline src/style.css:12]: This color is default
```suggestion
```

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,78 @@
...
+  white-space: nowrap;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+
+  transition: color 0.3s;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```
Comment (volodymyr-soltys97) [inline src/style.css:62]: Create a variable for repeated color and use it everywhere

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,78 @@
...
+}
+
+.nav__link.is-active {
```
Comment (volodymyr-soltys97) [inline src/style.css:65]:
```suggestion
.is-active {
```

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,78 @@
...
+  color: #00acdc;
+}
+
+.nav__link.is-active::after {
```
Comment (volodymyr-soltys97) [inline src/style.css:69]:
```suggestion
.is-active::after {
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's make your code better

File: src/style.css (hunk 1 above)
Comment (Oleh88777) [reply src/style.css:12]: Fixed

File: src/style.css (hunk 2 above)
Comment (Oleh88777) [reply src/style.css:62]: Fixed

File: src/style.css (hunk 3 above)
Comment (Oleh88777) [reply src/style.css:65]: Fixed

File: src/style.css (hunk 4 above)
Comment (Oleh88777) [reply src/style.css:69]: Fixed

**Review by anastasiiavorobiova — APPROVED** [review]

**Pull request closed without merging**